src/components/ProductTours/_ProductTours.scss

```diff
@@ -15,3 +15,11 @@
     }
   }
 }
+
+.pgn__checkpoint {
+  border-top: 8px solid #D23228;
```

Comment:
Same here, there's no exact match for this in paragon but we could use one of the danger colors from here, like $danger-500? https://paragon-openedx.netlify.app/foundations/colors/